src/utils/settingsValidation/index.ts

```diff
@@ -108,6 +111,10 @@ export function settingsValidation(config: unknown, validationParams: ISettingsV
   // creates a settings object merging base, defaults and config objects.
   const withDefaults = merge({}, base, defaults, config) as ISettingsInternal;
 
+  // ensure a valid logger
+  const log = logFactory('splitio'); // @ts-ignore
```

Comment:
Maybe we can try to add a comment with the reason each time we need a @ts-ignore